docs/worker/methods/stop.md

```diff
@@ -53,4 +55,13 @@ def stop(
     ):
         if isinstance(exception, pymongo.errors.ServerSelectionTimeoutError):
             self.stop()
+
+    def on_stop(
+        self,
+        task,
+    ):
+        super().on_stop(
```

Comment:
There is no need to call the parent's `on_stop` as it is just an empty stub.

Reply:
not exactly.. It raise an `WorkerStop` exception which stop the worker

Reply:
WDYM? the base on_stop is just an empty stub. What am I missing?